bundles/model/org.eclipse.smarthome.model.rule/src/org/eclipse/smarthome/model/rule/jvmmodel/RulesJvmModelInferrer.xtend

```diff
@@ -114,7 +114,7 @@ class RulesJvmModelInferrer extends ScriptJvmModelInferrer {
                     logger.warn("Duplicate field: '{}'. Ignoring '{}'.", name, type.class.name)
                 }
             ]
-
+            
```

Comment:
This change that only adds tailing whitespaces can be removed from the commit.

Reply:
removed